### DISCUSSION TITLE
Virtual garage

### DISCUSSION THREAD

**Discussion by Jaz0314** [Vdauphin/HeartsAndMinds#1607] [Question and answer] [answered]
Is there a way to make it where the team leader can us the garage without being an admin?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Yes, https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf#L74
Write something like: 
```sqf
{(serverCommandAvailable "#logout" || !isMultiplayer || leader group player == player) and btc_p_garage}
```
But garage is a bit clunky :/

Cheers
